daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/LanguageVersion.scala

```diff
@@ -1,9 +1,8 @@
 // Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package com.daml.lf.language
-
-import com.daml.lf.VersionRange
+package com.daml.lf
+package language
```

Comment:
Scala question: What does it mean to have multiple package statements? What's the purpose?

Reply:
it means you can reference `com.daml.lf.X` without qualifying as well as `com.daml.lf.language.X`.

Reply:
Thanks! :-)